portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,6 +39,17 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Create the query and prepared query to load comment entities from database
+    Query query = new Query("Comment");
```

Comment:
Save this as a constant at the top of the file. Try to stay away from hardcoded values in code